README.md

```diff
@@ -1,3 +1,33 @@
+# Shoutem CLI
+
 Shoutem CLI is a command line tool which helps you build Shoutem extensions.
 
 You can read more about CLI in our [documentation](http://shoutem.github.io/docs/extensions/reference/cli)
+
+## Local Development
+
+For local development there are three ways of approaching this:
+
+#### 1. Compiling and running from the repository
+
+1. Run `npm i` inside repository
+2. Run `node path/to/build/shoutem.js <command>`, where `<command>` is the command you're testing
+
+After making changes to the code, you can run `rm -rf build && npm run build` to recompile and then continue using step 2 described above.
+
+#### 2. Editing global node_modules
+
+While messy and harder to keep track of changes (since you ), this has proven effective. You can make changes in your global `node_modules` directory on your machine in the `src` directory, then re-build the CLI using `npm run build` within the `cli` directory (where the package.json is).
```

Comment:
`(since you )` - this, I think, should be deleted

Reply:
Whoops, forgot to finish that sentence.